README.rst

```diff
@@ -135,9 +135,10 @@ installation.
 
 0.  Make sure PostgreSQL is configured to allow WAL archival and retrieval.
     ``postgresql.conf`` should have ``wal_level`` set to ``archive`` or
-    higher and ``max_wal_senders`` set to a non-zero value, for example::
+    higher and ``max_wal_senders`` set to at least ``1`` (``archive_command`` mode)
+    or at least ``2`` (``pg_receivexlog`` mode), for example::
 
-        wal_level = archive
+        wal_level = hot_standby
```

Comment:
this doesn't now match our requirement above, and there's no real reason we need hot_standby.